src/xrGame/UIGameCustom_script.cpp

```diff
@@ -13,6 +13,8 @@ SCRIPT_EXPORT(CUIGameCustom, (CDialogHolder), {
                          .def("wnd", &StaticDrawableWrapper::wnd),
 
         class_<CUIGameCustom, CDialogHolder>("CUIGameCustom")
+            .def("TopInputReceiver", &CUIGameCustom::TopInputReceiver)
+            .def("SetMainInputReceiver", &CUIGameCustom::SetMainInputReceiver)
```

Comment:
This is wrong place for this export.

1. It should be exported here: https://github.com/OpenXRay/xray-16/blob/dev/src/xrGame/ui/UIWindow_script.cpp#L26
2. Maybe we also should export `SetTopInputReceiver`...

Reply:
1. Moved
2. What's `SetTopInputReceiver`? Or did you mean rename `SetMainInputReceiver` to `SetTopInputReceiver`?

Reply:
I mean exporting additional function :)
```cpp
.def("SetMainInputReceiver", &CUIGameCustom::SetMainInputReceiver)
.def("SetTopInputReceiver", &CUIGameCustom::SetMainInputReceiver)
```

Reply:
Applied your suggestion